alchemy/test/aws/ec2/internet-gateway-attachment.test.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  DescribeInternetGatewaysCommand,
+  EC2Client,
+} from "@aws-sdk/client-ec2";
+import { describe, expect } from "vitest";
+import { alchemy } from "../../../src/alchemy.js";
+import { InternetGatewayAttachment } from "../../../src/aws/ec2/internet-gateway-attachment.js";
```

Comment:
make sure to use `.ts` when importing. run `bun check` (biome check) which should catch this